docs/sphinx/scripting.rst

```diff
@@ -120,7 +120,9 @@ without necessarily having to lock scripts/interpreters every frame.
 
 .. py:function:: timer_add(callback, milliseconds)
 
-    Adds an timer callback which triggers every *milliseconds*.
+    Adds an timer callback which triggers every *milliseconds*. (Note:
+    Using instance methods as callbacks is not supported. Always use
+    module methods.)
```

Comment:
I would rather lose the parenthetical and just start a new line or new paragraph with "Note:".

Reply:
Updated to put remove parentheses. Waiting on CI builds to confirm the newline / new paragraph.